src/vs/workbench/parts/emmet/node/emmetActions.ts

```diff
@@ -261,17 +273,15 @@ export abstract class EmmetEditorAction extends EditorAction {
 
 export class BasicEmmetEditorAction extends EmmetEditorAction {
 
-	private emmetActionName: string;
-
 	constructor(id: string, label: string, alias: string, actionName: string, kbOpts?: ICommandKeybindingsOptions) {
 		super({
 			id: id,
 			label: label,
 			alias: alias,
 			precondition: EditorContextKeys.Writable,
-			kbOpts: kbOpts
+			kbOpts: kbOpts,
+			actionName: actionName
```

Comment:
We can use here ES2015 object destructuring for `id`, `label`, `alias`, `kbOpts`, `actionName`?